docs/tools/android_lint.md

```diff
@@ -0,0 +1,7 @@
+# Android Lint
```

Comment:
I would add a warning that before the Android Gradle plugin v3.1 (not sure what exact canary version), Lint cannot find issues in Kotlin files. It's not _our_ fault, but I imagine some may come reporting that as an issue.